### ISSUE TITLE
Show style for active tab

### ISSUE THREAD

**Issue by chriskuech** [ViktorQvarfordt/vscode-pitch-black-theme#2]
It is hard to tell which tab is in focus.  Please add a subtle active tab style.  I currently override this theme with

```json
      "tab.activeBorderTop": "#333"
```

**Comment by ViktorQvarfordt** [comment]
Good point, I agree that it's hard to see which tab is in focus. I consider making the font color of the active tab brighter or bold. Border is also interesting. But really want to keep low clutter.

**Comment by chriskuech** [comment]
Ya I didn't realize that foregroundColor is font color.  I would actually prefer something like

```
"tab.activeForeground": "#9cdcfe"
```

These are the editable fields that I see on active tab
```
tab.activeForeground
tab.activeBackground
tab.activeBorder
tab.activeBorderTop
```

**Comment by ViktorQvarfordt** [comment]
I tried some variations and ended up with simply reducing the brightness of inactive tabs (and keeping active tab at max brightness). This was the cleanest solution. It is now easy to distinguish the active tab from the other tabs.

**Comment by ViktorQvarfordt** [comment]
Merged in 6304f76d2f94cb32bc8ba066c13e652848495a9c.
